framework/OpenMod.Runtime/Runtime.cs

```diff
@@ -186,10 +183,10 @@ private void SetupSerilog()
             if (!File.Exists(loggingPath))
             {
                 // First run, logging.yml doesn't exist yet. We can not wait for auto-copy as it would be too late.
-                using Stream stream = typeof(AsyncHelper).Assembly.GetManifestResourceStream("OpenMod.Core.logging.yml");
-                using StreamReader reader = new StreamReader(stream);
+                using var stream = typeof(AsyncHelper).Assembly.GetManifestResourceStream("OpenMod.Core.logging.yml");
+                using var reader = new StreamReader(stream ?? throw new InvalidOperationException());
```

Comment:
It would be better if you throw something like "new Exception("Couldn't find resource: OpenMod.Core.logging.yml") instead.